projects/m1/013-birth-date-to-astrological-sign/python/main.py

```diff
@@ -0,0 +1,29 @@
+#ask to user a day and a month
+day = int(input('type a day: '))
+month = input('type a month: ').lower()
+
+# if else statements that returns the zodiacal sign by comparing the day typed by user with a range of numbers and the month typed with a string
+if day in range(22,32) and month == 'december' or day in range(1,20) and month == 'january':
+    print('your sign is capricorn')
+elif day in range(20,32) and month == 'january' or day in range(1,19) and month == 'febraury':
+    print('your sign is Acquarius')
```

Comment:
It might be a good idea to save the zodiac sign in a variable and print it at the end _(so you call the_ `print` _function once)_